ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestrouting.py

```diff
@@ -163,4 +163,4 @@ def add_verify_peers_test(description: str, afi: str, bgp_neighbor_ip: str, safi
             elif neighbor_peer_group_type == "evpn":
                 add_verify_peers_test(description="bgp evpn peer state established (evpn)", afi="evpn", bgp_neighbor_ip=bgp_neighbor_ip)
 
-        return {self.anta_module: anta_tests}
+        return {self.anta_module: anta_tests} if anta_tests.get("bgp") else None
```

Comment:
BGP stuff?

Reply:
This is a fix on something already implemented.